core/trino-main/src/test/java/io/trino/sql/planner/iterative/rule/TestSimplifyExpressions.java

```diff
@@ -69,6 +71,28 @@ public void testPushesDownNegations()
         assertSimplifies("NOT (X IS DISTINCT FROM Y)", "NOT (X IS DISTINCT FROM Y)");
     }
 
+    @Test
+    public void testSimplifyMatchAllWildCardLikePredicate()
```

Comment:
This method is misnamed. It includes tests where there are no wildcards in the pattern. Let's rename it to "testLikeExpressions"